web-gateway/src/main/twirl/views/logout.scala.html

```diff
@@ -0,0 +1,5 @@
+@(implicit nav: Nav)
+
+@main("Logout") {
+  <h3>You have been successfully logged out.</h3>
+}
```

Comment:
Do we need an extra page? Could we go redirect to the main screen with a cleared `ctx`?

Reply:
@ignasi35  I have now redirected to the index page